RomanNumeralsConverter.Tests/ValidationRulesTests/NoThreeSameNumeralsOtherThanMInARowValidatorTests.cs

```diff
@@ -0,0 +1,27 @@
+using FluentAssertions;
+using NUnit.Framework;
+using RomanNumeralsConverter.ValidationRules;
+
+namespace RomanNumeralsConverter.Tests.ValidationRulesTests
+{
+  internal class NoThreeSameNumeralsOtherThanMInARowValidatorTests
+  {
+    private NoThreeSameNumeralsOtherThanMInARowValidator _subject;
+
+    [SetUp]
+    public void SetUp()
+    {
+      _subject = new NoThreeSameNumeralsOtherThanMInARowValidator();
+    }
+
+    [TestCase("XXXX")]
+    public void Should_ReturnThreeNumeralsInARowError_When_MoreThanThreeNumeralsOtherThanMInARow(string romanNumeral)
```

Comment:
I missing here test case for opposite scenario(what happen if it is not three numerals ...)